demos/gaze_estimation_demo/cpp_gapi/src/custom_kerenels.cpp

```diff
@@ -0,0 +1,200 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "kernel_packages.hpp"
+#include "custom_kernels.hpp"
+
+#include <opencv2/imgproc.hpp>
+
+using namespace gaze_estimation;
+
+struct Estimators {
+    std::shared_ptr<HeadPoseEstimator>  headPoseEstimator;
+    std::shared_ptr<LandmarksEstimator> landmarksEstimator;
+    std::shared_ptr<EyeStateEstimator>  eyeStateEstimator;
+    std::shared_ptr<GazeEstimator>      gazeEstimator;
+};
+
+Estimators estimators;
+
+void rotateImageAroundCenter(const cv::Mat& srcImage,
+                             const float angle,
+                                   cv::Mat& dstImage) {
+    const auto width = srcImage.cols;
+    const auto height = srcImage.rows;
+
+    const cv::Size size(width, height);
+
+    const cv::Point2f center(static_cast<float>(width / 2), static_cast<float>(height / 2));
+
+    const auto rotMatrix = cv::getRotationMatrix2D(center, static_cast<double>(angle), 1);
+    cv::warpAffine(srcImage, dstImage, rotMatrix, size, 1, cv::BORDER_REPLICATE);
+}
+
+inline void toCHW(const cv::Mat& src, cv::Mat& dst) {
+    dst.create(cv::Size(src.cols, src.rows * src.channels()), CV_32F);
+    std::vector<cv::Mat> planes;
+    for (int i = 0; i < src.channels(); ++i) {
+        planes.push_back(dst.rowRange(i * src.rows, (i + 1) * src.rows));
+    }
+    cv::split(src, planes);
+}
+
+void preprocessing(const cv::Mat& src, const cv::Size& new_size, const float roll, cv::Mat& dst) {
+    cv::Mat rot, cvt, rsz;
+    rotateImageAroundCenter(src, roll, rot); // rotate
+    cv::resize(rot, rsz, new_size);          // resize
+    rsz.convertTo(cvt, CV_32F, 1.f);         // convert to F32
```

Comment:
```suggestion
    rsz.convertTo(cvt, CV_32F);              // convert to F32
```
You are not interested in scaling so don't mention it at all.

Reply:
Removed.